### ISSUE TITLE
Crossmark Widget accessibility issues (WCAG 2.0 Levels A and AA)

### ISSUE THREAD

**Issue by robtree** [CrossRef/crossmark#23]
http://crossmark.crossref.org/widget/v2.0/readme.html

1. It is not possible to open the Crossmark dialog by using keyboard control only (WCAG 2.1.1, see https://www.w3.org/TR/WCAG20-TECHS/G202.html)
2. The 'Check for updates' button does not have a text alternative (WCAG 1.1.1, see https://www.w3.org/TR/WCAG20-TECHS/H37.html)
3. The modal dialog does not receive keyboard focus when opened (WCAG 2.4.3, see https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/SCR37.html)
4. The modal dialog does not retain keyboard focus when cycling through its contents using the Tab key
5. The modal dialog cannot be closed by pressing the Escape key.

The WAI-ARIA Authoring Practices 1.1 produced by the W3C Working Group provides an example (3.8) of an accessible modal dialog implementation.

https://www.w3.org/TR/wai-aria-practices/#dialog_modal

Example:

https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html
